src/Api/Controllers/Payload/Requests/Users/GetAllSharedUsersOfASharedEntryPaginatedQueryParameters.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Api.Controllers.Payload.Requests.Users;
+
+public class GetAllSharedUsersFromASharedEntryPaginatedQueryParameters : PaginatedQueryParameters
```

Comment:
If not use sort then dont extend this class, copy and paste the page and size properties